packages/cli-tool/src/commands/generate/index.ts

```diff
@@ -0,0 +1,87 @@
+import * as fs from 'node:fs'
+
+import {Command} from '@oclif/core'
+import {cli} from 'cli-ux'
+import Inquirer from 'inquirer'
+
+import getChoices from '../../helpers/choices'
+import {formatHookErrorMsg, hookFailed} from '../../helpers/hook-error'
+
+const VERSION_CONTROL_OPTIONS: { [key: string]: string } = {
```

Comment:
Should we move this const and `setVersionControl` function into `versionControl.ts` file?
Then this file won't get fat

Reply:
Moving options into a separate files is another argument in favor of exported functions (instead of hooks) :D

Reply:
moved in [fb959b5](https://github.com/nimblehq/react-templates/pull/95/commits/fb959b5bf65c08d34212162339ca1393c93ed045)